data.sql

```diff
@@ -10,4 +10,70 @@ INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_k
 INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES(8, 'Angemon', '2005-06-12', 1, TRUE, -45);
 INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES(9, 'Bourman', '2005-06-07', 7, TRUE, 20.4);
 INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES(10, 'Blossom', '1995-10-13', 3, TRUE, 17);
-INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES(11, 'Bitto', '2022-05-14', 4, TRUE, 22);
\ No newline at end of file
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES(11, 'Bitto', '2022-05-14', 4, TRUE, 22);
+
+-- Update insert data into the owners table
+INSERT INTO owners(full_name, age) VALUES('Sam Smith', 34);
+INSERT INTO owners(full_name, age) VALUES('Jennifer Orwell', 19);
+INSERT INTO owners(full_name, age) VALUES('Bob', 45);
+INSERT INTO owners(full_name, age) VALUES('Melody Pond', 77);
+INSERT INTO owners(full_name, age) VALUES('Dean Winchester', 14);
+INSERT INTO owners(full_name, age) VALUES('Jodie Whittaker', 38);
+
+-- Update insert data into the species table
+INSERT INTO species(name) Values('Pokemon');
+INSERT INTO species(name) Values('Digimon');
+
+-- Modify your inserted animals so it includes the species_id value:
+-- Start the transaction for modifying the species_id column in animals table
+BEGIN;
+
+-- If the name ends in "mon" it will be Digimon
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+
+-- All other animals are Pokemon
+UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+
+-- Update commit the transaction to make sure it persists
+COMMIT;
+
+-- Modify your inserted animals to include owner information (owner_id)
+-- Update start the transaction for adding owner id to animals table
+Begin;
+
+-- Update Sam Smith owns Agumon
+UPDATE animals SET owner_id = 1 WHERE name LIKE 'Agumon';
+
+-- Update Jennifer Orwell owns Gabumon and Pikachu
+-- Gabumon
+UPDATE animals SET owner_id = 2 WHERE name LIKE 'Gabumon';
+
+-- Pikachu
+UPDATE animals SET owner_id = 2 WHERE name LIKE 'Pikachu';
+
+-- Update Bob owns Devimon and Plantmon
+-- Devimon
+UPDATE animals SET owner_id = 3 WHERE name LIKE 'Devimon';
+
+-- Plantmon
+UPDATE animals SET owner_id = 3 WHERE name LIKE 'Plantmon';
+
+-- Melody Pond owns Charmander, Squirtle, and Blossom.
+-- Charmander
+UPDATE animals SET owner_id = 4 WHERE name LIKE 'Charmander';
+
+-- Squirtle
+UPDATE animals SET owner_id = 4 WHERE name LIKE 'Squirtle';
+
+-- Blossom
+UPDATE animals SET owner_id = 4 WHERE name LIKE 'Blossom';
+
+-- Dean Winchester owns Angemon and Boarmon
+-- Angemon
+UPDATE animals SET owner_id = 5 WHERE name LIKE 'Angemon';
+
+-- Boarmon
+UPDATE animals SET owner_id = 5 WHERE name LIKE 'Bourman';
```

Comment:
Please correct the typo here. It should be `Boarmon` instead of `Bourman`.

Reply:
Thank you @SajjadAhmad14 for this review.
You are very keen thank you I will change the typo right away!